### ISSUE TITLE
suspended (tty input)

### ISSUE THREAD

**Issue by lights0123** [r-darwish/topgrade#298]
When running topgrade `v3.7.1` on Arch Linux, Konsole, and `zsh`, I occasionally get the output `[1]  + 243237 suspended (tty output)  topgrade`, where I have to run `fg` to get it to continue. I suspect that this is related to the [changes in arguments to `zsh`](https://github.com/r-darwish/topgrade/compare/v3.7.0...v3.7.1)—although it also occured after Flatpakl. Here's a sample output:

```
 10:53:28 PM   benschattinger  ~/Documents  
❱ topgrade       

―― 22:54:38 - System update ――――――――――――――――――――――――――――――――――――――――――――――――――――
:: Synchronizing package databases...
 core is up to date
 extra                                                                                          1644.9 KiB  5.23 MiB/s 00:00 [###########################################################################] 100%
 community                                                                                         4.8 MiB  28.6 MiB/s 00:00 [###########################################################################] 100%
 multilib is up to date
:: Starting full system upgrade...
 there is nothing to do

―― 22:54:57 - zplug ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
[zplug] Start to update 4 plugins in parallel

 ✔  Updated!              agkozak/zsh-z
 ✔  Updated!              zsh-users/zsh-syntax-highlighting
 ✔  Updated!              zsh-users/zsh-autosuggestions
 ✔  Up-to-date            eendroroy/alien

[zplug] Elapsed time: 0.8515 sec.
 ==> Updating finished successfully!

―― 22:54:59 - oh-my-zsh ――――――――――――――――――――――――――――――――――――――――――――――――――――――――
[1]  + 243237 suspended (tty input)  topgrade
 149  10:54:59 PM   benschattinger  ~/Documents  
❱ fg         
[1]  + 243237 continued  topgrade
Updating Oh My Zsh
remote: Enumerating objects: 9, done.
remote: Counting objects: 100% (9/9), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 5 (delta 3), reused 3 (delta 1), pack-reused 0
Unpacking objects: 100% (5/5), 709 bytes | 354.00 KiB/s, done.
From https://github.com/ohmyzsh/ohmyzsh
 * branch              master     -> FETCH_HEAD
   78b07e92..aaa87063  master     -> origin/master
Updating 78b07e92..aaa87063
Fast-forward
 plugins/pyenv/pyenv.plugin.zsh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)
Current branch master is up to date.
         __                                     __   
  ____  / /_     ____ ___  __  __   ____  _____/ /_                                                                                                                                                            
 / __ \/ __ \   / __ `__ \/ / / /  /_  / / ___/ __ \                                                                                                                                                           
/ /_/ / / / /  / / / / / / /_/ /    / /_(__  ) / / /                                                                                                                                                           
\____/_/ /_/  /_/ /_/ /_/\__, /    /___/____/_/ /_/                                                                                                                                                            
                        /____/                                                                                                                                                                                 
Hooray! Oh My Zsh has been updated and/or is at the current version.
To keep up on the latest news and updates, follow us on twitter: https://twitter.com/ohmyzsh
Get your Oh My Zsh swag at: https://shop.planetargon.com/collections/oh-my-zsh

―― 22:55:04 - rustup ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
info: syncing channel updates for 'stable-x86_64-unknown-linux-gnu'
info: syncing channel updates for 'nightly-x86_64-unknown-linux-gnu'
  stable-x86_64-unknown-linux-gnu unchanged - rustc 1.40.0 (73528e339 2019-12-16)
   nightly-x86_64-unknown-linux-gnu updated - rustc 1.42.0-nightly (3291ae339 2020-01-15) (from rustc 1.42.0-nightly (0de96d37f 2019-12-19))

info: cleaning up downloads & tmp directories

―― 22:55:33 - Cargo ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
    Updating registry 'https://github.com/rust-lang/crates.io-index'
(omitted)
Updated 3 packages.
Package  Installed                                 Latest                                    Needs update
xargo    d1d7606528ffd7709f8bda365fb3b676fc4f5a19  5984b8d3e8fe5405761117a227aa905e87f9f036  Yes
ddh      a55235e573283b2dcb1fbdc8ac7ce9c10773248d  a55235e573283b2dcb1fbdc8ac7ce9c10773248d  No

Updated 1 git package.
Overall updated 4 packages: cargo-make, cargo-nono, du-dust, xargo.

―― 22:57:55 - Yarn ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
warning @vue/cli > @vue/cli-ui > vue-cli-plugin-apollo > apollo > git-parse > babel-polyfill > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning @vue/cli > @vue/cli-ui > vue-cli-plugin-apollo > apollo > git-parse > babel-polyfill > babel-runtime > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning @vue/cli-init > vue-cli@2.9.6: This package has been deprecated in favour of @vue/cli

―― 22:58:12 - RubyGems ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――
Updating installed gems
Updating io-console
Fetching io-console-0.5.4.gem
WARNING:  You don't have /home/benschattinger/.gem/ruby/2.7.0/bin in your PATH,
          gem executables will not run.
Building native extensions. This could take a while...
Gems updated: io-console paint rdoc rspec rspec-core rspec-expectations rspec-mocks rspec-support term-ansicolor sync tins

―― 22:58:34 - Flatpak User Packages ――――――――――――――――――――――――――――――――――――――――――――
Looking for updates…
F: Warning: Treating remote fetch error as non-fatal since runtime/org.freedesktop.Platform/x86_64/1.6 is already installed: Unable to load summary from remote gnome: Error resolving “sdk.gnome.org”: Name or service not known
F: Warning: Can't find runtime/org.freedesktop.Platform.GL.default/x86_64/1.6 metadata for dependencies: Unable to load metadata from remote gnome: summary fetch error: Error resolving “sdk.gnome.org”: Name or service not known
[1]  + 243237 suspended (tty output)  topgrade
 150  10:58:35 PM   benschattinger  ~/Documents  
❱ fg
[1]  + 243237 continued  topgrade
Nothing to do.
Looking for updates…
Required runtime for com.leinardi.gwe/x86_64/stable (runtime/org.gnome.Platform/x86_64/3.34) found in remote flathub


        ID                                                    Branch             Op            Remote                Download
 1. [✓] org.freedesktop.Platform.GL.default                   19.08              u             flathub                 2.2 MB / 90.9 MB
 2. [✓] org.freedesktop.Platform.GL.nvidia-430-26             1.4                u             flathub                70.9 MB / 71.3 MB
 3. [✓] org.freedesktop.Platform.GL.nvidia-430-40             1.4                u             flathub                70.6 MB / 71.0 MB
(rest omitted)
```

**Comment by r-darwish** [comment]
Please check if this still happens with Togprade 3.7.2

**Comment by lights0123** [comment]
Fixed with v3.7.2.
